code/pizzabot.js

```diff
@@ -1,7 +1,95 @@
+
+// *** declairing variables
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const pizzaPrice = 80;
+
+const pizzaVegImg = document.getElementById("veg");
+const pizzaHawImg = document.getElementById("haw");
+const pizzaPepImg = document.getElementById("pep");
+
+
+// !! HIDING THIS PART SINCE IT ONLY WAS PART OF ITERATION 1-7. !!
+/*** Function 1 - Order Name 
+    const validateOrderName = (order) => {
+        if (order === vegetarian || order === hawaiian || order === pepperoni) {
+          return true;
+        } else {
+          return false;
+        }
+      };
+***/
+
+// *** Function 2 - Total Cost
+const calculateTotalCost = (quantity, price) => {
+  return quantity * price;
+}
+
+// *** Function 3 - Cooking Time
+const calculateCookingTime = (time) => {
+  if (time <= 2) {
+    return 10;
+  } else if (time >= 3 && time <=5) {
+    return 15;
+  } else {
+    return 20;
+  }
+}
+
+// *** Function - Submit form
+
+const handleSubmit = (event) => {
+  event.preventDefault();
+  let userName = document.forms.pizzaForm.userName.value;
+  // makes first letter to uppercase and the rest of the mane to lowercase
+  userName = userName.charAt(0).toUpperCase() + userName.slice(1).toLowerCase();
+
+  const pizzaType = document.forms.pizzaForm.elements.pizza.value;
+  const orderQuantity = document.forms.pizzaForm.numberOfPizzas.value;
+  
+    const orderTotal = calculateTotalCost(orderQuantity, pizzaPrice);
+    const cookingTime = calculateCookingTime(orderQuantity);
+
+    // 'ternary operator' (puts the 's for plural number of pizzas)
+    const plural = orderQuantity > 1 ? "'s" : "";
+
+    document.getElementById(
+      "orderSummery"
+    ).innerHTML = `Hello ${userName}! I'll get started on your ${orderQuantity} ${
+      pizzaType + plural
+    } right away. It will cost ${orderTotal} kr and will take ${cookingTime} minutes.`;
+    
+    if (pizzaType === "Vegetarian Pizza") {
+      pizzaVegImg.setAttribute ("style", "display:block;");
```

Comment:
You have access to values `"Vegetarian Pizza"` and others inside variables defined at the top of the file - let's use them instead of actual strings.

Imagine situation in which you have many conditionals like this (let's say 5) and in each one of them somewhere you check whether `pizza === 'Vegetarian Pizza'`. 4 out 5 times you wrote it well, but for the 5th time you forgot to add `' Pizza'` and end up with conditional `pizza === 'Vegetarian'`. Or your pizza menu changed and you need to go over all 5 occurrences and change it there.

To avoid these kind of situations, convention says that we should use ONLY variables everywhere. Thanks to that we are sure that all of these variable occurrences always has the same value - value that is being stored inside variable :)